### ISSUE TITLE
Cannot find module 'loader-utils'

### ISSUE THREAD

**Issue by jdb8** [mxmul/closure-loader#10]
Hello, thanks for this project. I wanted to try running the example code but can't get past the following error:

``` shell
$ npm install
...
$ npm run build

> closure-loader-example@1.0.0 build /Users/jb/code/closure-loader/examples/es6-closure-lib
> webpack

Hash: defc3590a0fcc5535c4b
Version: webpack 1.12.13
Time: 2358ms
     Asset       Size  Chunks             Chunk Names
    app.js    2.36 kB       0  [emitted]  app
app.js.map    2.16 kB       0  [emitted]  app
index.html  323 bytes          [emitted]
    + 2 hidden modules

ERROR in Cannot find module 'loader-utils'
 @ ./src/app.js 5:46-101
```

I've tried installing loader-utils explicitly, but it still can't be found. It's possible I'm doing something silly here, or perhaps something's changed with one of the libraries since the examples were written? I couldn't find anything helpful on the first couple of pages of Google.

**Comment by eXaminator** [comment]
Hi jdb8,

I'll have a look at it sometime today after work! :)

**Comment by eXaminator** [comment]
I can't recreate this error.

Can you give me some additional info?
1. What node and npm version are you using?
2. Can you see if npm installed `loader-utils` either in the examples `node_modules` or `node_modules/closure-loader/node_modules`?

**Comment by jdb8** [comment]
Ah, I believe I've just solved this while attempting to provide the info!

I hadn't considered that running `npm install` inside the examples folder was not enough - I also needed to run it inside the closure-loader folder itself, to generate the `node_modules` folder (containing `loader-utils`) there too. I assume this is because by default node/webpack is scanning up the tree to find an installation of closure-loader, and uses the one in `../..`? I had assumed that it was a dependency inside the example's package.json, but I see now that this is not the case.
